<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogExport</class>
 <widget class="QDialog" name="DialogExport">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exporter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icones/icon</normaloff>:/icones/icon</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushTick">
       <property name="text">
        <string>Tout cocher</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushUntick">
       <property name="text">
        <string>Tout décocher</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="listPresets">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>120</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushFolder">
       <property name="maximumSize">
        <size>
         <width>26</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../ressources.qrc">
         <normaloff>:/icones/folder</normaloff>:/icones/folder</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Emplacement</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="comboFormat">
       <item>
        <property name="text">
         <string notr="true">.sf2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">.sf3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">.sfz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineFolder"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QCheckBox" name="checkGM">
       <property name="text">
        <string>classement général midi</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QCheckBox" name="checkBank">
       <property name="text">
        <string>un répertoire par banque</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QCheckBox" name="checkPreset">
       <property name="text">
        <string>numéro de preset en préfixe</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="comboQuality">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Haute</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moyenne</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Basse</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelQuality">
       <property name="text">
        <string>Qualité</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushExport">
       <property name="text">
        <string>Exporter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushTick</tabstop>
  <tabstop>pushUntick</tabstop>
  <tabstop>lineFolder</tabstop>
  <tabstop>comboFormat</tabstop>
  <tabstop>pushAnnuler</tabstop>
  <tabstop>pushExport</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
